### ISSUE TITLE
Does OpenComputers support HD fonts from resource packs?

### ISSUE THREAD

**Issue by ghost** [MightyPirates/OpenComputers#750]
After debugging ComputerCraft and coming to the conclusion that I can't make Faithful 32 font look proper without rewriting the mod, I just want to know how well OC handles different fonts before I look into adding support.

**Comment by Wuerfel21** [comment]
No, its fixed to unifont to allow asian people that somehow got the mod to do... stufff

**Comment by Techokami** [comment]
There is a config option to change between using unifont and a font sheet from the textures. But the font sheet is slower and does not have full Unicode support.

Sent from my Windows Phone

**Comment by fnuecke** [comment]
What Techokami said, basically. The texture based font renderer isn't _much_ slower, though. At least not by a degree where it should weigh in when you're already throwing higher resolution textures at Minecraft ;-)

To change the renderer used, look for the `fontRenderer` setting in the config. The used texture is located at [`assets/opencomputers/textures/font`](https://github.com/MightyPirates/OpenComputers/tree/master-MC1.7.10/src/main/resources/assets/opencomputers/textures/font). Note the `chars.txt` file which contains the list characters provided by the font texture, as well as the size of the individual characters in pixels (`width height`). You can override these files using resource packs.

For completion's sake: you _can_ also override the character definitions used for the dynamic font renderer by overriding the [`assets/opencomputers/unifont.hex`](https://github.com/MightyPirates/OpenComputers/blob/master-MC1.7.10/src/main/resources/assets/opencomputers/unifont.hex) file, which is also possible via resource packs. The sizes of the individual characters is fixed here, however, and you must use the "binary" format of that file, which will prove to be a major pain unless you can find some good tool to aid you in converting a font texture to that format, I guess.

TL;DR: you can probably do what you want by switching to the texture based font renderer :-)
